Tests/SwiftParserTest/Assertions.swift

```diff
@@ -434,7 +434,18 @@ func assertDiagnostic<T: SyntaxProtocol>(
       }
     }
   }
-  if let fixIts = spec.fixIts {
+
+  if diag.fixIts.count != (spec.fixIts?.count ?? 0) {
```

Comment:
Should we just make `spec.fixIts` non-optional and default to `[]`? That way we don’t need to handle with optionals here.